x-pack/metricbeat/module/googlecloud/loadbalancing/_meta/docs.asciidoc

```diff
@@ -0,0 +1,32 @@
+Load Balancing Metricset to fetch metrics from https://cloud.google.com/load-balancing/[Load Balancing] in Google Cloud Platform.
+
+The `loadbalancing` Metricset contains all metrics exported from the https://cloud.google.com/monitoring/api/metrics_gcp#gcp-loadbalancing[Stackdriver API]. The field names have been left untouched for people already familiar with them.
+
+[float]
+=== Fields
+
+- `loadbalancing.https.backend_latencies`: A distribution of the latency calculated from when the request was sent by the proxy to the backend until the proxy received from the backend the last byte of response.
+- `loadbalancing.https.backend_request_bytes_count`: The number of bytes sent as requests from HTTP/S load balancer to backends.
+- `loadbalancing.https.backend_request_count`: The number of requests served by backends of HTTP/S load balancer.
+- `loadbalancing.https.backend_response_bytes_count`: The number of bytes sent as responses from backends (or cache) to HTTP/S load balancer.
+- `loadbalancing.https.frontend_tcp_rtt`: A distribution of the RTT measured for each connection between client and proxy.
+- `loadbalancing.https.backend_latencies`: A distribution of the latency calculated from when the request was sent by the proxy to the backend until the proxy received from the backend the last byte of response.
```

Comment:
nit: you can sort all fields for searching purposes

Reply:
Good idea, indeed. Sorted.